src/complexity.cc

```diff
@@ -163,25 +162,34 @@ std::vector<BenchmarkReporter::Run> ComputeStats(
     // We don't report aggregated data if there was a single run.
     return results;
   }
+
```

Comment:
i'm not sure how this method ended up here, but i'm wondering if it belongs in here, or in a new `statistics.cc` implementation file.